### PR TITLE
docs: add commit type

### PR DESCRIPTION
**Pull request by Insun35**
- Fix typo in folder name: developement -> development
- Add Commit Message Format in `convention.md`

### CODE REVIEW

**Review by chokobole — APPROVED** [review]
LGTM

**Review by TomTaehoonKim — APPROVED** [review]
LGTM

**Review by ashjeong — CHANGES_REQUESTED** [review]

File: docs/development/convention.md [hunk 1]
```diff
@@ -35,3 +35,14 @@ size_t size = size_t{1} << bit; // Recommended to cast as per the style guide
 - If a file contains device only codes, there will be suffixes `.cu.xx` despite it can be run on `ROCm`.
 - If a file contains host and device codes, then suffixes are not appended.
   For examples, see files under `tachyon/devices/gpu`.
+
+## Commit
+
+### Commit Type Classification
+
+There are some cases where it is ambiguous to classify the commit type. Here are the cases we follow:
```
[the remark this answers is not in the document]
Comment (ashjeong) [reply docs/development/convention.md:43]: Actually, I think that including the first sentence is useful in providing context for the section

The second sentence can be worded as you wish possibly as such (or mix-matched):
"Here are further clarifications on how to determine the proper commit type given previous cases we have come across."
"Here are further clarifications we follow to determine the proper commit type given previous cases encountered."
"Given previous ambiguous cases, we have adopted the following additional guidelines:"

**Review by ashjeong — APPROVED** [review]
LGTM

**Review by dongchangYoo — APPROVED** [review]
LGTM

**Pull request merged**